android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -735,18 +735,6 @@ private void showNotification(String id, int type, long timestamp, String[] args
                         break;
                 }
                 break;
-            case REWARDS_NOTIFICATION_NO_INTERNET:
```

Comment:
@Miyayes should we show different kind of notification when internet is not available and user tries to open rewards panel ?

Reply:
@emerick Do we show anything on desktop? Let's match desktop. My impression is that this "Uh oh" notification just doesn't exist anymore in general on desktop, so wanted to match.

Reply:
No, we don't do/show anything specific in that scenario on desktop.

Reply:
@deeppandya Okay, then good to remove without replacement. Thanks.

Reply:
Sounds good @Miyayes .